Devx 24n (#1000)
spectrocloud/librarium: content/docs/04.5-devx/05-app-profile/00-create-app-profile.md

 *  Database Services: Different database applications such as MongoDB, PostgreSQl, Redis, etc.

    <br />

    <InfoBox>

    The applications follows a hierarchy, where in the output variables values from the application added first can be passed to application tiers added after that. The reverse is not possible.

    To view the available output variables from the below tiers in the code editor, type in ```{{.spectro.}}```.
    </InfoBox>

When you select a service, you will see the configuration options for that service on the righthand side of the screen. Once you have provided all the required information for the service you may click on the **Review** button to proceed. You may also add additional services or tiers, by clicking on the blue **Actions** button on the top lefthand of the screen. 

<br />

5. **Review** the configuration and click on the **Finish Configuration** button.

The Palette Dev Engine App Profile is now created and and can be used for Apps deployment. To edit or delete the App Profile, navigate to the **App Profile** page. Select the desired App Profile and click on the  **Settings** button at the top righthand side. 

<br />

## Validation

> You can validate the App Profile is avaiable and ready for us by visiting the App Profiles page. In the App Profiles page, you will find your App Profile listed.

Reviewer: ```suggestion
You can validate the App Profile is available and ready for use by visiting the App Profiles page. On the App Profiles page, you will find your App Profile listed.
```

4. **Add Service**: Select a service to start configuring the App Profile from the list of services available. The service you select will require you provide information to generate the configuration. The following is an explanation of the types of services available. 

>   * [Container Deployment](/devx/app-profile/container-deployment): Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.

Reviewer: ```suggestion
  * Container Deployment: [Containers](https://www.docker.com/resources/what-container/) are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment. Review the [Palette Container Deployment](/devx/app-profile/container-deployment) to learn more about container deployment.
```
Re-adding

>  * [Helm](/devx/registries#palettehelmregistry): Palette App profile Pack from Helm registry charts. The public Palette Pack registry and a few popular Helm chart repositories are already available out of the box. Add additional pack registries or public or private chart registries to Palette.

Reviewer: ```suggestion
 * [Helm](/devx/registries/helm-registry#palettehelmregistry): Helm chart repositories are available for use. You can add additional Helm registries to Palette, visit the [Palette Helm Registry](/devx/registries/helm-registry#palettehelmregistry) documentation page for guidance.
```
Re-adding previous commit from feedback
